### PR TITLE
Add options to find query

### PR DESCRIPTION
**Pull request by lionelrudaz**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -23,18 +23,18 @@ class Service {
...
     let order = utils.getOrder(filters.$sort);
-    let query = {
+    let query = Object.assign({
       where, order,
       limit: filters.$limit,
       offset: filters.$skip,
       attributes: filters.$select || null
-    };
-    
+    }, params.sequelize);
```
Comment (ekryski) [inline src/index.js]: :+1: can we put this under `$sequelize` instead? That way it follows a similar pattern to the other special query string params?

**Comment by ekryski** [comment]
I'm cool with this. We should adds some docs around this as well. @feathersjs/core-team where do you want to put them? In the docs or leave something specific like this in the README? I'm inclined to put it in the docs so that it all stays together.

Refs #12.

**Comment by marshallswain** [comment]
Definitely in the docs repo.

File: src/index.js (hunk 1 above)
Comment (daffl) [inline src/index.js]: I think that's ok. It's part of `params` not `params.query` (where we use that pattern to somehow avoid property name clashes).

Comment (ekryski) [inline src/index.js]: ah right. My bad. ok let's :shipit:

**Comment by ekryski** [comment]
Ok then I'll create a corresponding issue in the `feathers-docs` repo.

**Comment by lionelrudaz** [comment]
Thanks guys. Let me know if I can be of any help.

**Comment by daffl** [comment]
Thank you for contributing! I just published a patch release with your fix.

**Comment by lionelrudaz** [comment]
You're far too kind. Thanks for your help.

**Pull request merged**